src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -986,19 +991,34 @@ function hideInvalidRecipientTypeOptions($giverSelect) {
     const giverType = $giverSelect.val();
     const $recipientSelect = $giverSelect.closest('.form_question').find('select[name="recipienttype"]');
     const recipientType = $recipientSelect.val();
+    const qnType = $giverSelect.closest('.form_question').find('[name="questiontype"]').val();
     switch (giverType) {
     case 'STUDENTS':
-        // all recipientType options enabled
+        /* all recipientType options enabled
+         * except that when the question type is Rank Recipient, SELF and NONE cannot be chosen
+         */
```

Comment:
Let's not use `/* .. */` style block comment. Just repeat the `//` on all lines.